docs/application/native/guides/location-sensors/device-sensors.md

```diff
@@ -409,6 +409,24 @@ The following table lists the measurement data that the HRM LED green sensor pro
 | Timestamp                        | `unsigned long long` | -                                       | Microseconds |
 | values[0]: HRM green light value | `int`                | Min. value = 0<br> Max. value = 4194304 | -            |
 
+<a name="hrm_green_batch"></a>
+## Heart Rate Monitor LED Green batch Sensor
```

Comment:
@bbrto21 Request you to kindly change all headings to sentence case. 
https://github.com/Samsung/tizen-docs/blob/master/styleguide/style.md#capitalization
Thanks

Reply:
@nawab-samsung I updated this PR to apply capitalization. Please review again